scala/build/maintenance.Dockerfile

```
@@ -116,22 +116,29 @@ RUN apt install -y \
 
 # Install Python and required packages
 # Optional: If you need venv for virtual environments
-RUN apt-get update && apt-get install -y software-properties-common && \
+RUN apt-get update && apt-get install -y software-properties-common curl && \
```

Hadolint

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)